global-state/jar/resources/log4j.properties

```diff
@@ -0,0 +1,12 @@
+# Root logger  set to INFO level of debugging.
+log4j.rootLogger=INFO, A1
+log4j.logger.com.sixsq.slipstream.globalstate=DEBUG
```

Comment:
This should not be DEBUG by default.  Probably should be INFO.

Reply:
DONE